pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -448,6 +448,19 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 		uniqueVolumeName = volumehelper.GetUniqueVolumeNameForNonAttachableVolume(volume.podName, plugin, volumeSpec)
 	}
 
+	if attachablePlugin != nil {
+		if isNotMount, mountCheckErr := rc.mounter.IsLikelyNotMountPoint(volume.mountPath); mountCheckErr != nil {
```

Comment:
@msau42 @jingxu97 @jsafrane - do you think that we need to use IsNotMountPoint here instead of IsLikelyNotMountPoint to avoid issues in the reconciler with respect to local volumes? For reference... https://github.com/kubernetes/kubernetes/pull/48402

Reply:
@ianchakeres thanks for bring this up. I think we do need to consider this . But since IsNotMountPoint will list all the mounts, to reduce the overhead, maybe we can cache the mounts?

Reply:
@jingxu97 This particular check is for only attachable volumes and isLikelyNotMountPoint should work for attachable volumes. Do we still need it?

Reply:
@chakri-nelluri  I first thought we should check the mount path for both attachable and non-attachable plugins. But in the current code workflow, if volume is not attachable, we only reconstruct if the the volume is not in desired state. So the reconstructed state will only be used by teardown operation (umount). So it is ok to not check the mount path exist or not. 
I am ok with this part of code. Thanks!